packages/flutter_markdown/test/custom_syntax_test.dart

```diff
@@ -58,6 +59,30 @@ void defineTests() {
         expect(span.recognizer.runtimeType, equals(TapGestureRecognizer));
       },
     );
+
+    testWidgets(
+      'replace word "container" with container widget',
```

Comment:
I feel like this summary doesn't accurately reflect what this test does, unless I'm reading the code wrong